docs/userguide/products/fault-association.html

```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>faultl-association Product Documentation</title>
+<link rel="stylesheet" href="../userguide.css" />
+</head>
+
+<body>
+<h1>fault-association Product Documentation</h1>
+<p><a href="index.html">&laquo; Back to Products Documentation</a></p>
+
+<h2>Types</h2>
+<dl>
+	<dt>fault-association</dt>
+	<dd>This product contains data about probabilities that a fault in the SCEC Community Fault Model
+		is associated with an event.</a>.</dd>
+</dl>
+
+<!--
+<h2>Sources</h2>
+<dl>
+	<dt>SCEC Community Fault Model</dt>
+	<dd>Harvard University, Dept of Earth & Planetary Sciences</dd>
+	<dd>Public Key</dd>
+</dl>
+-->
+
+<h2>Properties</h2>
+<dl>
+    <dd>Not applicable</dd>
+</dl>
+
+
+<h2>Contents</h2>
+<dl>
+	<dt>faultassociations.json</dt>
+	<dd>
+		[application/json] Structured JSON string containing fault
+		association probabilities.
+	</dd>
+</dl>
+
+<h2>JSON Format</h2>
+<p>The JSON file is an extension of GEOJSON standard and contains an array of a metadata object, a result object, and
```

Comment:
`array` is a little misleading.  I would say something like:

> <code>faultassociations.json</code> is a GeoJson FeatureCollection of fault associations, and includes additional <code>metadata</code> and <code>result</code> attributes that summarize inputs and results.

When referencing filenames or attributes/properties, I recommend using an html <code>code</code> element, for example:
```
<code>faultassociations.json</code>
```